### PR TITLE
Verbose error message for moving window functions

### PR DESCRIPTION
**Pull request by auguzun**
for moving window functions there will be verbose error if the window size is bigger than the metrics retention

### CODE REVIEW

File: pkg/parser/interface.go [hunk 1]
```diff
@@ -65,6 +65,8 @@ var (
 	ErrDifferentCountMetrics = ParseError("both arguments must have the same number of metrics")
 	// ErrInvalidArgumentValue is an eval error returned when a function received an argument that has the right type but invalid value
 	ErrInvalidArgumentValue = ParseError("invalid function argument value")
+	// ErrMovingWindowSizeLessThanRetention is an eval error returned when a function received a moving window size bigger than the metrics retention
+	ErrMovingWindowSizeLessThanRetention = ParseError("moving window is bigger than the metrics retention")
```
Comment (timtofan) [inline pkg/parser/interface.go:69]: I'd rephrase to `windowSize param exceed metric retention`

Comment (auguzun) [reply pkg/parser/interface.go]: thanks, fixed

**Review by azhiltsov — APPROVED** [review]
